user/mimiker.ld

```diff
@@ -0,0 +1,55 @@
+OUTPUT_ARCH(mips)
+ENTRY(main)
+
+PHDRS
+{
+    text   PT_LOAD FLAGS(5); /* read-only, executable */
+    data   PT_LOAD FLAGS(6); /* read-write */
+    rodata PT_LOAD FLAGS(4); /* read-only */
+    bss    PT_LOAD FLAGS(6); /* read-write */
+}
+SECTIONS
+{
+    .text 0x00001000:
```

Comment:
`mips-linux-gnu` toolchain places `.text` section at `0x0400000`. I guess that's a right choice.

```
PROVIDE (__executable_start = SEGMENT_START("text-segment", 0x0400000)); . = SEGMENT_START("text-segment", 0x0400000) + SIZEOF_HEADERS;
```